extensions/wikia/WikiFactory/Close/SpecialCloseWiki_body.php

```diff
@@ -230,8 +230,10 @@ private function doProcess() {
 	}
 
 	private function prefixMainDomain( $cityId ) {
-		$mainDomain = substr( WikiFactory::getVarValueByName("wgServer", $cityId), 7 );
-		if(!empty($mainDomain)) {
+		//local prefix to not accidentally mix it with global wgServer
+		$localWgServer = WikiFactory::getVarValueByName( "wgServer", $cityId );
+		if ( preg_match( "/^https?:\/\/([\w\-\.]+)/", $localWgServer, $matches ) ) {
```

Comment:
I thought we already talked about DRY ;)

Reply:
preg_match is faster